File: atinject-integration/src/main/resources/webapp/scripts/atinject/useraffinity/UserAffinityNotification.js
define(["atinject/WebSocketNotification"],
function(WebSocketNotification){
    function UserAffinityNotification() {
        this._class = "org.atinject.api.useraffinity.dto.UserAffinityNotification"
    };

    UserAffinityNotification.prototype = new WebSocketNotification();

        UserAffinityNotification.prototype.getUrl = function() {
        return this.url;
    };

    UserAffinityNotification.prototype.setUrl = function(url) {
        this.url = url;
        return this;
    };


    
    return UserAffinityNotification;
});

File: atinject-api/atinject-api-user-affinity/src/main/java/org/atinject/api/useraffinity/dto/UserAffinityNotification.java
package org.atinject.api.useraffinity.dto;

import org.atinject.core.websocket.dto.WebSocketNotification;

public class UserAffinityNotification extends WebSocketNotification
{
    private static final long serialVersionUID = 1L;

    private String url;
    
    public String getUrl()
    {
        return url;
    }

    public UserAffinityNotification setUrl(String url)
    {
        this.url = url;
        return this;
    }
    
}
